File: src/frontend/src/app/components/cadastro/unidade-negocio/model/unidade-negocio-dto.ts
export class UnidadeNegocioDTO {
  id: string;
  codigo: string;
  nome: string;
  descricao?: string;
  cnpj?: string;
  ativa: boolean;
  createdAt?: Date;
  updatedAt?: Date;
  createdBy?: string;
  updatedBy?: string;

  constructor(
    id: string,
    codigo: string,
    nome: string,
    descricao: string | undefined,
    cnpj: string | undefined,
    ativa: boolean,
    createdAt?: Date,
    updatedAt?: Date,
    createdBy?: string,
    updatedBy?: string
  ) {
    this.id = id;
    this.codigo = codigo;
    this.nome = nome;
    this.descricao = descricao;
    this.cnpj = cnpj;
    this.ativa = ativa;
    this.createdAt = createdAt;
    this.updatedAt = updatedAt;
    this.createdBy = createdBy;
    this.updatedBy = updatedBy;
  }
}

File: src/backend/src/main/java/br/com/grupopipa/gestaointegrada/cadastro/unidadenegocio/UnidadeNegocioDTO.java
package br.com.grupopipa.gestaointegrada.cadastro.unidadenegocio;

import java.time.LocalDateTime;
import java.util.UUID;

import br.com.grupopipa.gestaointegrada.core.dto.DTO;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class UnidadeNegocioDTO implements DTO {

    private UUID id;
    private String codigo;
    private String nome;
    private String descricao;
    private String cnpj;
    private Boolean ativa;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private String createdBy;
    private String updatedBy;
}
